Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppCategories/MSFT_IntuneAppCategories.schema.mof

```diff
@@ -0,0 +1,14 @@
+[ClassVersion("1.0.0.0"), FriendlyName("IntuneAppCategories")]
+class MSFT_IntuneAppCategories : OMI_BaseResource
+{
+    [Key, Description("The unique identifier for an entity. Read-only.")] String Id;
+    [Required, Description("The name of the app category.")] String DisplayName;
```

Comment:
```suggestion
    [Key, Description("The name of the app category.")] String DisplayName;
```

Reply:
Thank you Fabien for your comments. I tried to search for you on Microsoft Teams, but could not find you. Do you work in MSFT?  The UTs are still failing locally in Github as well. So I needed to have a quick call with you if possible. Locally I am seeing the below issue:



I tried the below cmdlets as well:

PS D:\2\cat> .\M365TenantConfig.ps1
PSDesiredStateConfiguration\Configuration : The term 'PSDesiredStateConfiguration\Configuration' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of
the name, or if a path was included, verify that the path is correct and try again.
At D:\2\cat\M365TenantConfig.ps1:9 char:1
+ Configuration M365TenantConfig
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (PSDesiredStateC...n\Configuration:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException



PS D:\2\cat> Import-DSCResource -?

Import-DSCResource : The term 'Import-DSCResource' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify

that the path is correct and try again.

At line:1 char:1

+ Import-DSCResource -?

+ ~~~~~~~~~~~~~~~~~~

   + CategoryInfo          : ObjectNotFound: (Import-DSCResource:String) [], CommandNotFoundException

Start-dscconfiguration  M365TenantConfig -Wait -verbose -force
Start-dscconfiguration : M365TenantConfig is not a valid directory.
At line:1 char:2
+  Start-dscconfiguration  M365TenantConfig -Wait -verbose -force



Import-Module C:\Users\kajalp\Documents\GitHub\Microsoft365DSC\Tests\TestHarness.psm1 -Force

No error and no output, so I assum TestHarness.psm1 is successfully imported.
Invoke-TestHarness

All UTs are failing



PS D:\2\cat>      Start-dscconfiguration      M365TenantConfig -Wait -verbose -force

Start-dscconfiguration : M365TenantConfig is not a valid directory.

At line:1 char:2

+ Start-dscconfiguration M365TenantConfig -Wait -verbose -force

+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

   + CategoryInfo          : NotSpecified: (:) [Start-DscConfiguration], ArgumentException

   + FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.DesiredStateConfiguration.Commands.StartDscConfigurationCommand

Reply:
@Kajalp1079 How does your `M365TenantConfig.ps1` look like? Otherwise we can't help you. What you can do is initiate an export via `Export-M365DSCConfiguration -Components @("IntuneAppCategories") -Path "D:\testbed" -ApplicationId $clientId -TenantId $tenantIdName -CertificateThumbprint $certThumbprint -Verbose`, this will create a M365TenantConfig.ps1 file in the directory D:\testbed which is properly formatted for you. 

Before running the export cmdlet, please make sure that the directory for the MSFT_IntuneAppCategories is copied to `C:\Program Files\WindowsPowerShell\Modules\Microsoft365DSC\1.24.904.1\DSCResources` followed by `Import-Module Microsoft365DSC -Force`.